src/getting-started/notebook-demo.rst

```diff
@@ -11,14 +11,37 @@ You can find these notebooks in the :file:`notebooks/notebook-demo` directory of
 Double click the directory, then double click on the notebook to open it.
 You can click **Run** → **Run all Cells** to run the entire notebook, or type :kbd:`shift`\ -\ :kbd:`return` to run one cell at a time.
 
-In the `terminal <https://jupyterlab.readthedocs.io/en/latest/user/terminal.html>`_, you can find ``notebook-demo `` in the ``~/notebooks`` directory.
+In the `terminal <https://jupyterlab.readthedocs.io/en/latest/user/terminal.html>`_, you can find ``notebook-demo`` in the ``~/notebooks`` directory.
 
 .. note::
 
    The :file:`notebooks/notebook-demo` directory is hosted at https://github.com/lsst-sqre/notebook-demo.
    Every time you log into the Notebook Aspect, the default branch (called ``prod``) is updated from GitHub.
    Avoid running :command:`git commit` on the ``prod`` branch.
 
+How the notebooks are kept up to date
+-------------------------------------
+
+The system tries to keep the demo notebooks up to date.
+It does this by using the ``git stash`` and ``git apply`` commands to try to save user changes and update the notebooks without affecting user activity.
```

Comment:
It's a little weird to have "you" in much of the document and "user" or "the user" in this section (and then "you" again at the very end).  Maybe make this consistent by using "your" and "you" throughout?

Reply:
Sorry.  I meant to say "you" everywhere